example/custom_configuration.cpp/custom_configuration.cpp

```diff
@@ -0,0 +1,184 @@
+//
+// Simple example to demonstrate custom configuration
+// (system and component id) in MAVSDK
```

Comment:
It's a good idea to add an example to show how this is used. However, it would probably be easier to understand if the example was specific (and not generic like this one). E.g. there would be an example for the companion computer case and the example would be to do something like avoidance, something that makes sense for a companion.

Reply:
@julianoes Okay, I will modify this to be a companion example which streams/logs telemetry, I'm working on avoidance but thats out of the scope of this example IMO. Once avoidance is merged, I could write another example which takes advantage of the custom configuration to make avoidance possible.

Reply:
Great :+1:.